.github/actions/checkout-pytorch/action.yml

```diff
@@ -40,6 +40,11 @@ runs:
         fi
         mkdir "${GITHUB_WORKSPACE}"
 
+        # Use all available CPUs for fetching
+        cd "${GITHUB_WORKSPACE}"
+        git config --global fetch.parallel 0
+        git config --global submodule.fetchJobs 0
```

Comment:
Actually, the submodule fetch jobs are mostly iobound, not CPU bound, so we could increase the parallelism further to shave off even more time.

Reply:
This is still strictly better. I'd recommend adding it.